doc/challenge.html

```diff
@@ -37,7 +37,7 @@ <h2>Boost Graph Library Challenge and To-Do Items</h2>
      <tt>mutable_queue.hpp</tt>, <tt>fibonacci_heap.hpp</tt>.
       Somehow merge implementation with Dietmer's heaps and queues.</li>
 
-   <li><tt>disjoint_sets</tt> (see <a href="disjoint_sets.html">)</li>
+   <li><tt>disjoint_sets</tt> (see <a href="disjoint_sets.html">Disjoint Sets</a>)</li>
```

Comment:
I'm fairly naive about HTML, so can you explain to me why there needs to be a `</a>`? I thought the opening `<a` was closed by the 'naked' `>`?